ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+if [ "${ADDITIONAL_WORKERS}" == "0" ]; then
+    echo "No additional workers requested"
+    exit 0
+fi
+
+if [ "${ADDITIONAL_WORKERS_DAY2}" != "true" ]; then
+    echo "Skipping as the additional nodes have been provisioned at installation time."
+    exit 0
+fi
```

Comment:
This is interesting... there isn't a way to spin up additional workers on demand, so this might not impact anything.